tools/depcheck/pkg/cmd/trace/pkg.go

```diff
@@ -0,0 +1,122 @@
+package trace
```

Comment:
This file looks good.  Needs a few tests and we can merge it I think.

Reply:
Thanks, will add tests